RELEASES.md

```diff
@@ -255,6 +255,11 @@ Unless explicitly stated here, components that are called out as unstable or
 not covered may change in a future minor version. Breaking changes to
 "unstable" components will be avoided in patch versions.
 
+Go client API stability includes the `client`, `defaults` and `version` package
+as well as all packages under `pkg`, `core`, `api` and `protobuf`.
```

Comment:
q: Why is `pkg` included here?  How is it part of the client API?

Reply:
`pkg` packages are imported by clients. I did look through all the packages and tried to move the `pkg` packages which aren't imported by clients into `internal`. Ideally we would have a better namespace than `pkg` but don't think anyone came up with a better solution.